lib/bolt/plan_creator.rb

```diff
@@ -51,7 +51,14 @@ def self.validate_input(project, plan_name)
       end
     end
 
-    def self.create_plan(plans_path, plan_name, is_puppet)
+    # Create a new plan from the plan templates based on which language the
+    # user configured, and whether the plan wraps a script.
+    # @param plans_path [string] The path to the new plan
```

Comment:
```suggestion
    #
    # @param plans_path [string] The path to the new plan
```

Just for consistency with other documented methods. 😬 Sorry for the nit.